components/library/libetpan/libetpan.p5m

```diff
@@ -0,0 +1,198 @@
+file path=usr/include/libetpan/mailstream_low.h
+file path=usr/include/libetpan/mailstream_socket.h
+file path=usr/include/libetpan/mailstream_ssl.h
+file path=usr/include/libetpan/mailstream_types.h
+file path=usr/include/libetpan/mailthread.h
+file path=usr/include/libetpan/mailthread_types.h
+file path=usr/include/libetpan/mboxdriver.h
+file path=usr/include/libetpan/mboxdriver_cached.h
+file path=usr/include/libetpan/mboxdriver_cached_message.h
+file path=usr/include/libetpan/mboxdriver_message.h
+file path=usr/include/libetpan/mboxdriver_types.h
+file path=usr/include/libetpan/mboxstorage.h
+file path=usr/include/libetpan/mhdriver.h
+file path=usr/include/libetpan/mhdriver_cached.h
+file path=usr/include/libetpan/mhdriver_cached_message.h
+file path=usr/include/libetpan/mhdriver_message.h
+file path=usr/include/libetpan/mhdriver_types.h
+file path=usr/include/libetpan/mhstorage.h
+file path=usr/include/libetpan/mime_message_driver.h
+file path=usr/include/libetpan/mmapstring.h
+file path=usr/include/libetpan/namespace.h
+file path=usr/include/libetpan/namespace_parser.h
+file path=usr/include/libetpan/namespace_sender.h
+file path=usr/include/libetpan/namespace_types.h
+file path=usr/include/libetpan/newsfeed.h
+file path=usr/include/libetpan/newsfeed_item.h
+file path=usr/include/libetpan/newsfeed_item_enclosure.h
+file path=usr/include/libetpan/newsfeed_types.h
+file path=usr/include/libetpan/newsnntp.h
+file path=usr/include/libetpan/newsnntp_socket.h
+file path=usr/include/libetpan/newsnntp_ssl.h
+file path=usr/include/libetpan/newsnntp_types.h
+file path=usr/include/libetpan/nntpdriver.h
+file path=usr/include/libetpan/nntpdriver_cached.h
+file path=usr/include/libetpan/nntpdriver_cached_message.h
+file path=usr/include/libetpan/nntpdriver_message.h
+file path=usr/include/libetpan/nntpdriver_types.h
+file path=usr/include/libetpan/nntpstorage.h
+file path=usr/include/libetpan/pop3driver.h
+file path=usr/include/libetpan/pop3driver_cached.h
+file path=usr/include/libetpan/pop3driver_cached_message.h
+file path=usr/include/libetpan/pop3driver_message.h
+file path=usr/include/libetpan/pop3driver_types.h
+file path=usr/include/libetpan/pop3storage.h
+file path=usr/include/libetpan/qresync.h
+file path=usr/include/libetpan/qresync_types.h
+file path=usr/include/libetpan/quota.h
+file path=usr/include/libetpan/quota_parser.h
+file path=usr/include/libetpan/quota_sender.h
+file path=usr/include/libetpan/quota_types.h
+file path=usr/include/libetpan/uidplus.h
+file path=usr/include/libetpan/uidplus_types.h
+file path=usr/include/libetpan/xgmlabels.h
+file path=usr/include/libetpan/xgmmsgid.h
+file path=usr/include/libetpan/xgmthrid.h
+file path=usr/include/libetpan/xlist.h
+file path=usr/lib/$(MACH64)/libetpan.a
+link path=usr/lib/$(MACH64)/libetpan.so target=libetpan.so.17.2.0
+link path=usr/lib/$(MACH64)/libetpan.so.17 target=libetpan.so.17.2.0
+file path=usr/lib/$(MACH64)/libetpan.so.17.2.0
+file path=usr/lib/libetpan.a
```

Comment:
Do we really need static libs? We usually deliver them only when there's no alternative dynamic library.